pkgs/applications/networking/p2p/pyrosimple/default.nix

```diff
@@ -4,9 +4,24 @@
 , nix-update-script
 , pyrosimple
 , testers
+, buildPythonPackage
+, bencode-py
+, APScheduler
```

Comment:
Pretty sure this should be `apscheduler`
Fixing this should fix the ofborg failure.